bucket/inkscape-dev.json

```
@@ -0,0 +1,51 @@
+{
+    "version": "1.3-dev_2022-07-24",
+    "description": "Professional vector graphics editor",
+    "homepage": "https://inkscape.org",
+    "license": "GPL-3.0-or-later",
+    "architecture": {
+        "64bit": {
+            "url": "https://gitlab.com/inkscape/inkscape/-/jobs/2762744060/artifacts/raw/build/inkscape-1.3-dev_2022-07-24_3f4f2d82-x64.7z",
+            "hash": "b2f6cf413840990b3f40ef2f19f34695b2ceb704466ac5a07d8a5833dcb1b7ea"
+        },
+        "32bit": {
+            "url": "https://gitlab.com/inkscape/inkscape/-/jobs/2731415286/artifacts/raw/build/inkscape-1.3-dev_2022-07-16_7f6c2629d-x86.7z",
+            "hash": "969c1026a76316455658c67e9a3e095f12d106cb59a3d3eff43cc1a6f5da94ee"
+        }
+    },
+    "extract_dir": "inkscape",
+    "bin": [
+        "bin\\inkscape.com",
+        "bin\\inkview.com"
+    ],
+    "shortcuts": [
+        [
+            "bin\\inkscape.exe",
+            "Inkscape Dev"
+        ]
+    ],
+    "checkver": {
+        "script": [
+            "$builds = 'x32', 'x64'",
+            "$urls =  @()",
+            "foreach ($build in $builds) {",
+            "    $base_url = 'https://alpha.inkscape.org/snapshots/latest_master_'+ $build + '.html'",
```

```suggestion
            "$urls =  @()",
            "'x32', 'x64' | ForEach-Object {",
            "    $base_url = 'https://alpha.inkscape.org/snapshots/latest_master_'+ $_ + '.html'",
```

I've test it and works fine.